src/scripts/main.js

```diff
@@ -1,3 +1,65 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('tbody');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const removeRow = document.querySelector('.remove-row');
+const appendRow = document.querySelector('.append-row');
+
+function examination(maxLength, button, maxOrMin) {
+  if (maxLength === maxOrMin) {
+    button.disabled = true;
+  } else {
+    button.disabled = false;
+  }
+}
+
+appendColumn.addEventListener('click', () => {
+  const coloms = [...document.querySelectorAll('tr')];
```

Comment:
`columns`, typo, not fixed, also check in other places. and it's not really column when you select `tr` (table row), it's row